roles/pulp_common/tasks/install_pip.yml

```diff
@@ -300,6 +300,9 @@
       when: item.value.source_dir is defined
       environment:
         SETUPTOOLS_USE_DISTUTILS: stdlib
+        COMPOSE_PROFILE: standalone
+        DEV_SOURCE_PATH: 'pulpcore:pulp_ansible:pulp_container:galaxy_ng:galaxy-importer'
+        LOCK_REQUIREMENTS: 0
```

Comment:
@rochacbruno we can make vars, so you can set them on your playbook. Maybe create a new var and use "or" at galaxy_ng
```python
is_on_dev_environment or "IS_PULP_DEV_ENV" in os.environ
```